### PR TITLE
[Unity] Silly tag cleanup

### PR DESCRIPTION
**Pull request by AliRKat**
br and nbsp tags are cleaned, some lines fixed

### CODE REVIEW

**Comment by AliRKat** [comment]
[[Unity, Doc] Clean up the unity doc from silly tags](https://trello.com/c/hL9F5URN/1268-unity-doc-clean-up-the-unity-doc-from-silly-tags)

File: unity/next.md [hunk 1]
```diff
@@ -501,19 +525,19 @@ Countly.Instance.Init(config);</code></pre>
   <span><strong>IOS:</strong> on pre-iOS7 devices, it will return a hash of the MAC address. On iOS7 devices, it will be</span>
 </p>
 <p>
-  <span><code class="java">UIDevice identifierForVendor</code> or, if that fails for any reason,</span>
+  <span><code class="java">IDevice identifierForVendor</code> or, if that fails for any reason,</span>
```
Comment (ijunaid) [inline unity/next.md:528]: is it a typo?

File: unity/next.md [hunk 2]
```diff
@@ -1209,51 +1278,39 @@ configuration.SetMetricOverride(customMetric);</code></pre>
...
-  - App version<br>
-  <span>- Locale identifier</span>
+  * If sessions are used then also device metrics are collected which contains:
+  - Device model - Screen resolution - Screen density - OS name - OS version -
+  App version <span>- Locale identifier</span>
 </p>
 <p>
-  <span></span><span>* When generating a device ID, if no custom ID is provided, the SDK will use:</span><br>
-  <span>- Android: md5 of ANDROID_ID&nbsp;<br></span><span>- iOS: I</span><span>t will be vendor id and </span><span>advertising id as a fallback<br></span><span>- Windows stores apps: It will be advertising id<br></span><span>- Windows Standalone: It will be hash from the concatenation of strings taken from computer system hardware classes.</span><span></span>
+  <span></span><span>* When generating a device ID, if no custom ID is provided, the SDK will use:</span>
```
Comment (ijunaid) [inline unity/next.md:1294]: @AliRKat also remove <span> silly tags :)

**Review by ijunaid — COMMENTED** [review]
Good work 👍 
- In some places we are using "<code class="java">" and at some places "<code class="!whitespace-pre hljs language-csharp">" which make it a little inconsistent.

- There are still some silly <span> tags are present :)

**Review by ijunaid — APPROVED** [review]
Overall looks good to me 👍 
I am gonna approve this PR.

Still have lots of span tags in there
Also contains two types of code tags
`<code class="!whitespace-pre hljs language-csharp">` and `<code class="!csharp">`

@AliRKat please remove the span tags and fix the code class things, create a separate PR for them.

**Pull request merged**